### DISCUSSION TITLE
Food Substitution Question

### DISCUSSION THREAD

**Discussion by junqi00** [magpiemodel/magpie#623] [Q&A] [answered]
Dear MAgPIE community,

I am currently exploring the MAgPIE model and want to study plant-based meat alternatives by MAgPIE. I noticed in Module 15 there's an option for the substitution of livestock products with plant-based products, but I am a bit unclear about the details of this substitution.

How does this substitution work within the model: Is it achieved by replacing livestock products with proteins directly from vegetables, fruits and legumes, or is there a specific category for plant-based meat alternatives? If the model currently does not have a specific setting for plant-based meat alternatives, could you kindly offer some guidance on which modules should I modify to include such options (like the scp which has been included in the model) ?

Thanks!
Junqi

**Comment by mscrawford** [comment, marked as answer]
Dear Junqi,

The most up-to-date documentation one could use for better understanding how the food scenarios current work would be in the Food System Transformation preprint, here: https://www.researchsquare.com/article/rs-2928708/v1. 

You can check out the configuration file  `scenario_fsec.csv` in `magpie/config` in particular to see what the most relevant flags are for a transition away from ruminants, in particular under the `ruminants` scenario (column). If you have substitution, the extra calories come from staples, such as cereals, roots, and tubers. You’ll be generally interesting in the 15_food module.

Best!
Michael

**Reply by junqi00** [reply]
Dear Michael,

Thank you very much for your detailed reply. I will carefully look into the suggestions and the files you mentioned.

Best regards,
Junqi
